app/src/main/res/layout/fragment_home.xml

```diff
@@ -19,141 +20,152 @@
-                    android:layout_marginStart="19dp"
-                    android:layout_marginTop="14dp"
-                    app:layout_constraintTop_toBottomOf="@id/my_health_description"
-                    app:layout_constraintStart_toEndOf="@id/my_health_icon"
-                    android:text="@string/home_my_health_card_check_in_button"
-                    android:onClick="@{() -> vm.onCheckInClick()}"
-                    android:textAppearance="@style/card_view_link"
-                    android:textColor="@color/colorAccent"
-                    android:textStyle="bold"/>
-            </androidx.constraintlayout.widget.ConstraintLayout>
+                    android:layout_marginTop="@dimen/default_line_height"
+                    android:layout_marginBottom="@dimen/default_line_height"
+                    android:text="@string/home_my_health_card_description"/>
+
+            </LinearLayout>
         </androidx.cardview.widget.CardView>
         <androidx.cardview.widget.CardView
             android:id="@+id/contact_alerts_card"
-            android:layout_width="344dp"
-            android:layout_height="144dp"
-            app:cardBackgroundColor="#FAFAFA"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            app:cardBackgroundColor="@android:color/white"
             app:layout_constraintTop_toBottomOf="@id/my_health_card"
             app:layout_constraintStart_toStartOf="@id/my_health_card"
             app:layout_constraintEnd_toEndOf="@id/my_health_card"
             app:cardCornerRadius="8dp"
-            android:layout_marginTop="33dp">
-            <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_marginTop="51dp"
+            android:onClick="@{() -> vm.onSeeAlertsClick()}">
+            <LinearLayout
+                android:orientation="vertical"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent">
-
-                <androidx.appcompat.widget.AppCompatImageView
-                    android:id="@+id/contact_alerts_icon"
+                <TextView
+                    android:id="@+id/alert_count"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    app:layout_constraintTop_toTopOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    android:layout_marginStart="16dp"
-                    android:layout_marginTop="18dp"
-                    android:src="@drawable/ic_contact_alert_icon" />
+                    android:layout_marginTop="@dimen/default_line_height"
+                    android:layout_marginStart="48dp"
+                    android:text="@{vm.title}"
+                    android:textSize="17sp"
+                    android:textColor="@color/red"
+                    android:visibility="@{vm.newAlerts ? View.VISIBLE : View.GONE}"
```

Comment:
You can use `isVisible` property to bind directly to a boolean (see `XmlBindingAdapters`).